app/src/main/java/de/mvhs/android/zeiterfassung/EditActivity.java

```diff
@@ -0,0 +1,30 @@
+package de.mvhs.android.zeiterfassung;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+public class EditActivity extends AppCompatActivity {
+
+  @Override
+  protected void onCreate( Bundle savedInstanceState ) {
+    super.onCreate( savedInstanceState );
+    setContentView( R.layout.activity_edit );
+    Toolbar toolbar = (Toolbar) findViewById( R.id.toolbar );
+    setSupportActionBar( toolbar );
+
+    FloatingActionButton fab = (FloatingActionButton) findViewById( R.id.fab );
```

Comment:
In der Hausaufgabenbeschreibung auf https://github.com/WebDucer-MVHS/kurs-android-I-a/wiki/Hausaufgabe-Tag-2 sehe ich aber auch keinen Bezug auf Buttons irgendeiner Art; entsprechend habe ich auch keine Verbaut. Da ich aber auch nicht weiss, was als nächstes auf dem Lehrplan steht, habe ich den Autogenerierten Code von Android Studio auch erstmal so stehen lassen. Wo/wie hätten die Buttons verbaut werden sollen?